server/src/assignment/assignment.service.ts

```diff
@@ -114,11 +114,16 @@ export function transformIntoAssignments(projects, contactid, recodedCbFullName)
       const actions = transformActions(project.dcp_dcp_project_dcp_projectaction_project);
       const milestones = project.dcp_dcp_project_dcp_projectmilestone_project;
       const dispositions = project.dcp_dcp_project_dcp_communityboarddisposition_project;
+      const participantRoleLookup = {
+        BP: 'Borough President',
+        BB: 'Borough Board',
+        CB: 'Community Board',
+      };
```

Comment:
Nit but I'd always keep lookups like this in the very top of the file and with ALL_CAPS:

```javascript
      const PARTICIPANT_ROLE_LOOKUP = {
        BP: 'Borough President',
        BB: 'Borough Board',
        CB: 'Community Board',
      };
```